DeliceLydia/Politico
# Title: #171792784 feedbacks

#### What does this PR do?
This PR is for feedbacks given last week and tests

#### Description of Task to be completed
 -worked on the feedbacks of the UI
 -started writing tests
 -added travis.yml file
 -wrote signup tests
 -wrote signin tests

#### How should this be manually tested?
-clone the project in your local machine
-cd in the project
-run npm install to update all packages used
-run index.html if you want to view UI(user interface)
-run npm start if you want to test endpoints on postman
-run npm test if you want to see the coverage

 #### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
(https://www.pivotaltracker.com/story/show/171792784)[#171792784]

#### Screenshots (if appropriate)
<img width="744" alt="Screen Shot 2020-03-17 at 13 05 02" src="https://user-images.githubusercontent.com/44745855/76850412-ffb55380-684f-11ea-9042-46e38f4203d6.png">
